tools/cbmc/proofs/HTTP/IotHttpsClient_ReadContentLength/IotHttpsClient_ReadContentLength_harness.c

```diff
@@ -0,0 +1,16 @@
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "FreeRTOS_Sockets.h"
+
+/* FreeRTOS+TCP includes. */
+#include "iot_https_client.h"
+#include "iot_https_internal.h"
+
+#include "../global_state_HTTP.c"
+
+void harness() {
+	IotHttpsResponseHandle_t respHandle = newIotResponseHandle();
+  	uint32_t * pContentLength = malloc(sizeof(uint32_t));
```

Comment:
Is there a reason to do this and not just declare "uint32_t contentLength" and pass "&contentLength"?  Just curious.  Probably doesn't make much difference.

Reply:
There is no reason. I changed it accordingly.